src/controller/CommissioneeDeviceProxy.h

```diff
@@ -105,6 +105,11 @@ class CommissioneeDeviceProxy : public DeviceProxy, public SessionDelegate
      */
     void CloseSession();
 
+    /**
+     *  Releases the underlying session (if any) from this proxy, without expiring it.
+     */
+    chip::Optional<SessionHandle> ReleaseSecureSession();
```

Comment:
I think this might be clearer as ForgetSecureSession, and document that it makes this proxy hand over its session to the caller and forget about it, without expiring it.

Reply:
Hm, "release" is what this is called in unique_ptr. Not sure about "forget"... maybe "detach"?